lib/extension.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+// The name of the package in order to give the Antora logger a useful name
+const { name: packageName } = require('../package.json')
+const generateIndex = require('./generate-index')
+
+function register (pipeline, {
+  config: { indexLatestOnly, languages, ...extraArguments }
+}) {
+  const logger = pipeline.require('@antora/logger').get(packageName)
+
+  if (Object.keys(extraArguments).length > 0) {
+    const args = Object.keys(extraArguments)
+      .map((x) => `"${x}"`)
+      .join(', ')
+    throw new Error(`Remove unrecognised extension option(s) for ${packageName}: ${args}`)
```

Comment:
Please use American English spelling in error messages, which means change "unrecognised" to "unrecognized".

Reply:
Oof, that stings. However, fair enough.